Documentation/operations/upgrade.rst

```diff
@@ -351,6 +351,9 @@ Annotations:
   ``policy_regeneration_time_stats_seconds`` have been deprecated in favor of
   ``endpoint_regenerations_total`` and
   ``endpoint_regeneration_time_stats_seconds``, respectively.
+* KVStoreMesh is now enabled by default in Clustermesh.
+  If you want to disable the KVStoreMesh, set Helm value ``clustermesh.apiserver.kvstoremesh.enabled=false``
```

Comment:
```suggestion
  If you want to disable KVStoreMesh, set Helm value ``clustermesh.apiserver.kvstoremesh.enabled=false``
```

Nit. It seems more natural to me (same below), but I'm definitely not a native speaker.

Reply:
I don't know, for me it's more natural to have "the" :D

Reply:
+1 to dropping "the", it sounds a bit off to me. We also reference it as just "KVStoreMesh" in the previous sentence, and at the end of intro.rst:
> for instructions on how to disable KVStoreMesh.